.github/workflows/CICD.yml

```diff
@@ -521,6 +521,35 @@ jobs:
         n_fails=$(echo "$output" | grep "^FAIL:\s" | wc --lines)
         if [ $n_fails -gt 0 ] ; then echo "::warning ::${n_fails}+ test failures" ; fi
 
+  test_freebsd:
+    runs-on: macos-latest
+    name: Tests/FreeBSD test suite
+    env:
+      # https://github.com/rust-lang/rustup/issues/2774
+      RUSTUP_IO_THREADS : 1
+      mem: 2048
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install/setup prerequisites
+      id: test
+      uses: vmactions/freebsd-vm@v0.1.4
+      with:
+        usesh: true
+        prepare: pkg install -y curl gmake
+        run: |
+          curl https://sh.rustup.rs -sSf --output rustup.sh
+          sh rustup.sh -y --profile=minimal
+    - name: "`make build`"
+      shell: bash
+      run: |
+        source $HOME/.cargo/env
+        cargo build
+    - name: "`make test`"
+      shell: bash
+      run: |
+        source $HOME/.cargo/env
+        cargo test -p uucore -p coreutils
```

Comment:
We should add `--features feat_os_unix` so that unix-specific utilities are tested as well

Reply:
yeah, indeed! I replicated what we had in cirrus. I am worried that it will surface some other issues but fine :)